### ISSUE TITLE
;j not only closes the downloads shelf, but also purges all downloads history (my history is gone :( )

### ISSUE THREAD

**Issue by quetzalcoatl** [brookhong/Surfingkeys#1408]
## Prelude
* Could you find answer in below documents?

    1. [FAQ](https://github.com/brookhong/Surfingkeys/wiki/FAQ) / [常见问题](https://github.com/brookhong/Surfingkeys/wiki/%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98) - NO
    2. [README](https://github.com/brookhong/Surfingkeys/blob/master/README.md) / [读我](https://github.com/brookhong/Surfingkeys/blob/master/README_CN.md) - NO

* Have you searched your problem in [issues](https://github.com/brookhong/Surfingkeys/issues)? - YES, I've found issue #556 that is identical to my problem, but that issue was just closed with no feedback
 
## Error details

Exactly as in #556: when I press ;j, the downloads shelf is closed as expected. However, also whole downloads history is cleared and all pending downloads are aborted. I think I also saw once that it also removed some downloaded files, but I might have misremembered it.

Originally, I'm a cVim user. I used the combination of gj very often, in SurfingKeys it doesn't exist, ok, and the ;j is the closest thing, even described as "close downloads shelf". Close. Not purge.

Why #556 was closed with no information? Is that a bug pending? or thought to be already fixed? Or is this behavior an intended feature?

![image](https://user-images.githubusercontent.com/1564880/106357004-ca98df00-6303-11eb-867e-f3869cffb01e.png)

SurfingKeys: 0.9.69

Browser: Chrome 86.0.4240.198 x64; Windows 10 Pro 20H2 Build:19042.630 FP:120.2212.31.0

**Comment by brookhong** [comment]
Yes, `;j` will clear download history by default, please see https://github.com/brookhong/Surfingkeys/blob/3a2b56ee1b7f539514022de36c29c00212ddff7b/pages/default.js#L627-L629

You could add below in your setting snippet to make it not clear history:

    mapkey(';j', '#12Close Downloads Shelf', function() {
        RUNTIME("closeDownloadsShelf");
    });

**Comment by quetzalcoatl** [comment]
Thank you! That fixes it for me!

However, please consider adding information about clearing the history on the help screen that displays the keyboard controls, I mean this:

![image](https://user-images.githubusercontent.com/1564880/106360939-7f8ac600-631b-11eb-92f5-78828c89730e.png)

I perceive clearing user's history as a quite drastic action, especially when unaware. Imagine "close bookmark panel" that would helpfully erase all bookmarks. Alright, it may be helpful by default for some people, I won't suggest changing it, but there should be some warning :)
